etl/steps/data/garden/health/2022-12-28/deaths_karlinsky.meta.yml

```diff
@@ -1,5 +1,4 @@
-dataset:
-  sources:
+dataset: {}
```

Comment:
Hmm, is this the official way to say: "Nothing new to add in dataset metadata"?

Reply:
Same as https://github.com/owid/etl/pull/768#discussion_r1070967733, perhaps it'd be better to have `dataset` optional so that we don't have to write empty fields in yaml?